mgmt/rest/rest_v1_test.go

```diff
@@ -46,8 +46,8 @@ import (
 	"github.com/intelsdi-x/snap/core"
 	"github.com/intelsdi-x/snap/core/cdata"
 	"github.com/intelsdi-x/snap/core/ctypes"
-	"github.com/intelsdi-x/snap/mgmt/rest/fixtures"
-	"github.com/intelsdi-x/snap/mgmt/rest/rbody"
+	"github.com/intelsdi-x/snap/mgmt/rest/v1/fixtures"
+	"github.com/intelsdi-x/snap/mgmt/rest/v1/rbody"
```

Comment:
In order to differentiate v1 and v2 dependencies / testing, I moved fixtures and rbody under the folder v1.

v2 has the same dependencies but I renamed it to mock / response.